.github/workflows/pipeline-dito.yml

```diff
@@ -0,0 +1,38 @@
+name: PipelineDiTo
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [main]
+    paths:
+      - "packages/dito/**"
+      - ".github/workflows/**"
+      - "!.github/workflows/shared.yml"
```

Comment:
This file does not exist, does it? Should be `!.github/workflows/pipeline-shared.yml`

Reply:
Same in other pipelines